scripts/yq_utils.sh

```diff
@@ -0,0 +1,15 @@
+# yq-go eats up blank lines
+# this function gets around that using diff with --ignore-blank-lines
+yq_ibl()
+{
+  set +e
+  diff_out=$(diff -B <(yq '.' "$2") <(yq "$1" "$2"))
+  error=$?
```

Comment:
this is not right, error may not be caught

```suggestion
  diff_out=$(diff -B <(yq '.' "$2") <(yq "$1" "$2")) || error=$?
```